### PR TITLE
Fixed typos in SwiftBasedManifestFormat.md

### PR DESCRIPTION
**Pull request by natan**
Fixes a few typos in SwiftBasedManifestFormat.md.

### CODE REVIEW

File: Documentation/Internals/SwiftBasedManifestFormat.md [hunk 1]
```diff
@@ -113,16 +113,16 @@ for target in package.targets {
...
+To that end, the declarative package specification portion of the file is "Swift" in the same sense that "JSON is Javascript". The syntax itself is valid, executable, Swift but the tools that process it will only accept a restricted, declarative, subset of Swift which can be statically evaluated, and which can be unambiguously, automatically rewritten by editing tools. We do not intend to define a "standard" syntax for "Swift Object Notation", but we intend to accept a natural restriction of the language which only accepts literal expressions. We do intend to allow the restricted subset to take full advantage of Swift's rich type inference and literal convertable design to allow for a succinct, readable, and yet expressive syntax.
```
Comment (mxcl) [inline Documentation/Internals/SwiftBasedManifestFormat.md]: What changed here? Whitespace? If so can we remove this (and the others) from the diff?

Comment (natan) [inline Documentation/Internals/SwiftBasedManifestFormat.md]: 'staticly' became 'statically'.

Comment (natan) [inline Documentation/Internals/SwiftBasedManifestFormat.md]: I recognize the diff is tough to read on GitHub. Would it be helpful for me to annotate all the updates with comments?

Comment (mxcl) [inline Documentation/Internals/SwiftBasedManifestFormat.md]: K, my bad, github bug: it wasn't highlighting the change.

**Pull request merged**